### ISSUE TITLE
Spotitube - Will not get Spotify lists

### ISSUE THREAD

**Issue by Rouzax** [kodinerds/repo#14]
I'm having issues when trying to load playlists from Spotify.

```
14:00:27 T:5464   ERROR: DXGI_ERROR_INVALID_CALL
14:00:53 T:5428   ERROR: Previous line repeats 4 times.
14:00:53 T:5428   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <class 'urllib2.URLError'>
                                            Error Contents: <urlopen error [Errno 1] _ssl.c:510: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure>
                                            Traceback (most recent call last):
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py", line 799, in <module>
                                                listSpotifyPlaylists(url)
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py", line 331, in listSpotifyPlaylists
                                                content = cache(url, 1)
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py", line 678, in cache
                                                content = opener.open(url).read()
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 410, in open
                                                response = meth(req, response)
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 523, in http_response
                                                'http', request, response, code, msg, hdrs)
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 442, in error
                                                result = self._call_chain(*args)
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 382, in _call_chain
                                                result = func(*args)
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 629, in http_error_302
                                                return self.parent.open(new, timeout=req.timeout)
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 404, in open
                                                response = self._open(req, data)
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 422, in _open
                                                '_open', req)
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 382, in _call_chain
                                                result = func(*args)
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 1222, in https_open
                                                return self.do_open(httplib.HTTPSConnection, req)
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 1184, in do_open
                                                raise URLError(err)
                                            URLError: <urlopen error [Errno 1] _ssl.c:510: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure>
                                            -->End of Python script error report<--
14:00:54 T:5464   ERROR: XFILE::CDirectory::GetDirectory - Error getting plugin://plugin.video.spotitube/?limit&mode=listSpotifyPlaylists&type&url=http%3a%2f%2fapi.tunigo.com%2fv3%2fspace%2ftoplists%3fregion%3dnl%26page%3d0%26per_page%3d50%26platform%3dweb
14:00:54 T:5464   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.spotitube/?limit&mode=listSpotifyPlaylists&type&url=http%3a%2f%2fapi.tunigo.com%2fv3%2fspace%2ftoplists%3fregion%3dnl%26page%3d0%26per_page%3d50%26platform%3dweb) failed
14:01:01 T:4748   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <class 'urllib2.URLError'>
                                            Error Contents: <urlopen error [Errno 1] _ssl.c:510: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure>
                                            Traceback (most recent call last):
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py", line 799, in <module>
                                                listSpotifyPlaylists(url)
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py", line 331, in listSpotifyPlaylists
                                                content = cache(url, 1)
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py", line 678, in cache
                                                content = opener.open(url).read()
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 410, in open
                                                response = meth(req, response)
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 523, in http_response
                                                'http', request, response, code, msg, hdrs)
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 442, in error
                                                result = self._call_chain(*args)
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 382, in _call_chain
                                                result = func(*args)
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 629, in http_error_302
                                                return self.parent.open(new, timeout=req.timeout)
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 404, in open
                                                response = self._open(req, data)
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 422, in _open
                                                '_open', req)
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 382, in _call_chain
                                                result = func(*args)
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 1222, in https_open
                                                return self.do_open(httplib.HTTPSConnection, req)
                                              File "C:\Program Files (x86)\Kodi\system\python\Lib\urllib2.py", line 1184, in do_open
                                                raise URLError(err)
                                            URLError: <urlopen error [Errno 1] _ssl.c:510: error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure>
                                            -->End of Python script error report<--
14:01:01 T:5464   ERROR: XFILE::CDirectory::GetDirectory - Error getting plugin://plugin.video.spotitube/?limit&mode=listSpotifyPlaylists&type&url=http%3a%2f%2fapi.tunigo.com%2fv3%2fspace%2ftoplists%3fregion%3dnl%26page%3d0%26per_page%3d50%26platform%3dweb
14:01:01 T:5464   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.spotitube/?limit&mode=listSpotifyPlaylists&type&url=http%3a%2f%2fapi.tunigo.com%2fv3%2fspace%2ftoplists%3fregion%3dnl%26page%3d0%26per_page%3d50%26platform%3dweb) failed
14:01:04 T:5464   ERROR: CScriptInvocationManager::ExecuteAsync - Not executing non-existing script script.favourites
14:01:04 T:5464   ERROR: Unable to activate the previous window
14:01:04 T:5464   ERROR: CScriptInvocationManager::ExecuteAsync - Not executing non-existing script script.favourites
14:01:10 T:5464   ERROR: Control 50 in window 10502 has been asked to focus, but it can't
14:01:13 T:6732   ERROR: Unable to find plugin
14:01:13 T:5464   ERROR: XFILE::CDirectory::GetDirectory - Error getting plugin://
14:01:13 T:5464   ERROR: CGUIMediaWindow::GetDirectory(plugin://) failed
```

**Comment by nickshe89** [comment]
Working in kodi 17

**Comment by Rouzax** [comment]
But that is stil in beta.

**Comment by Rouzax** [comment]
With the latest beta (Kodi 17 Beta 6) using the install from the Windows store it still crashed.
```
14:22:10.950 T:5224   DEBUG: CGUIMediaWindow::GetDirectory (plugin://plugin.video.spotitube/?content_type=video)
14:22:10.950 T:5224   DEBUG:   ParentPath = [plugin://plugin.video.spotitube/?content_type=video]
14:22:10.982 T:5940   DEBUG: CAddonDatabase::SetLastUsed[plugin.video.spotitube] took 31 ms
14:22:10.982 T:608   DEBUG: XFILE::CPluginDirectory::StartScript - calling plugin Youtube Music('plugin://plugin.video.spotitube/','1','?content_type=video')
14:22:10.982 T:2848   DEBUG: Thread LanguageInvoker start, auto delete: false
14:22:10.983 T:2848    INFO: initializing python engine.
14:22:11.083 T:2848   DEBUG: CPythonInvoker(1, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): start processing
14:22:11.180 T:2848   DEBUG: -->Python Interpreter Initialized<--
14:22:11.180 T:2848   DEBUG: CPythonInvoker(1, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): the source file to load is "C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py"
14:22:11.180 T:2848   DEBUG: CPythonInvoker(1, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): setting the Python path to C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\DLLs;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\Lib;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\python27.zip;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\lib\plat-win;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\lib\lib-tk;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\lib\site-packages
14:22:11.180 T:2848   DEBUG: CPythonInvoker(1, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): entering source directory C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube
14:22:11.180 T:2848   DEBUG: CPythonInvoker(1, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): instantiating addon using automatically obtained id of "plugin.video.spotitube" dependent on version 2.1.0 of the xbmc.python api
14:22:11.451 T:5224   DEBUG: ------ Window Init (DialogBusy.xml) ------
14:22:11.494 T:2848    INFO: CPythonInvoker(1, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): script successfully run
14:22:11.500 T:2848    INFO: Python script stopped
14:22:11.500 T:2848   DEBUG: Thread LanguageInvoker 2848 terminating
14:22:11.513 T:4348   DEBUG: Thread BackgroundLoader start, auto delete: false
14:22:11.528 T:4348   DEBUG: Thread BackgroundLoader 4348 terminating
14:22:11.537 T:5224   DEBUG: ------ Window Deinit (DialogBusy.xml) ------
14:22:12.993 T:5224   DEBUG: CInputManager::ProcessMouse: trying mouse action leftclick
14:22:13.005 T:5224   DEBUG: CGUIMediaWindow::GetDirectory (plugin://plugin.video.spotitube/?limit&mode=spotifyMain&type&url)
14:22:13.005 T:5224   DEBUG:   ParentPath = [plugin://plugin.video.spotitube/?content_type=video]
14:22:13.024 T:608   DEBUG: CAddonDatabase::SetLastUsed[plugin.video.spotitube] took 18 ms
14:22:13.024 T:5940   DEBUG: XFILE::CPluginDirectory::StartScript - calling plugin Youtube Music('plugin://plugin.video.spotitube/','2','?limit&mode=spotifyMain&type&url')
14:22:13.025 T:4020   DEBUG: Thread LanguageInvoker start, auto delete: false
14:22:13.025 T:4020    INFO: initializing python engine.
14:22:13.025 T:4020   DEBUG: CPythonInvoker(2, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): start processing
14:22:13.123 T:4020   DEBUG: -->Python Interpreter Initialized<--
14:22:13.123 T:4020   DEBUG: CPythonInvoker(2, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): the source file to load is "C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py"
14:22:13.123 T:4020   DEBUG: CPythonInvoker(2, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): setting the Python path to C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\DLLs;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\Lib;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\python27.zip;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\lib\plat-win;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\lib\lib-tk;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\lib\site-packages
14:22:13.123 T:4020   DEBUG: CPythonInvoker(2, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): entering source directory C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube
14:22:13.123 T:4020   DEBUG: CPythonInvoker(2, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): instantiating addon using automatically obtained id of "plugin.video.spotitube" dependent on version 2.1.0 of the xbmc.python api
14:22:13.508 T:5224   DEBUG: ------ Window Init (DialogBusy.xml) ------
14:22:13.532 T:4020    INFO: CPythonInvoker(2, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): script successfully run
14:22:13.544 T:4020    INFO: Python script stopped
14:22:13.544 T:4020   DEBUG: Thread LanguageInvoker 4020 terminating
14:22:13.550 T:4048   DEBUG: Thread BackgroundLoader start, auto delete: false
14:22:13.563 T:5224   DEBUG: ------ Window Deinit (DialogBusy.xml) ------
14:22:13.571 T:4048   DEBUG: Thread BackgroundLoader 4048 terminating
14:22:14.793 T:5224   DEBUG: CInputManager::ProcessMouse: trying mouse action leftclick
14:22:14.795 T:5224   DEBUG: CGUIMediaWindow::GetDirectory (plugin://plugin.video.spotitube/?limit&mode=listSpotifyPlaylists&type&url=http%3A%2F%2Fapi.tunigo.com%2Fv3%2Fspace%2Ftoplists%3Fregion%3Dus%26page%3D0%26per_page%3D50%26platform%3Dweb)
14:22:14.795 T:5224   DEBUG:   ParentPath = [plugin://plugin.video.spotitube/?limit&mode=spotifyMain&type&url]
14:22:14.803 T:5940   DEBUG: CAddonDatabase::SetLastUsed[plugin.video.spotitube] took 8 ms
14:22:14.803 T:608   DEBUG: XFILE::CPluginDirectory::StartScript - calling plugin Youtube Music('plugin://plugin.video.spotitube/','3','?limit&mode=listSpotifyPlaylists&type&url=http%3A%2F%2Fapi.tunigo.com%2Fv3%2Fspace%2Ftoplists%3Fregion%3Dus%26page%3D0%26per_page%3D50%26platform%3Dweb')
14:22:14.804 T:560   DEBUG: Thread LanguageInvoker start, auto delete: false
14:22:14.804 T:560    INFO: initializing python engine.
14:22:14.804 T:560   DEBUG: CPythonInvoker(3, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): start processing
14:22:14.904 T:560   DEBUG: -->Python Interpreter Initialized<--
14:22:14.904 T:560   DEBUG: CPythonInvoker(3, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): the source file to load is "C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py"
14:22:14.904 T:560   DEBUG: CPythonInvoker(3, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): setting the Python path to C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\DLLs;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\Lib;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\python27.zip;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\lib\plat-win;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\lib\lib-tk;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\lib\site-packages
14:22:14.904 T:560   DEBUG: CPythonInvoker(3, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): entering source directory C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube
14:22:14.905 T:560   DEBUG: CPythonInvoker(3, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): instantiating addon using automatically obtained id of "plugin.video.spotitube" dependent on version 2.1.0 of the xbmc.python api
14:22:15.296 T:5224   DEBUG: ------ Window Init (DialogBusy.xml) ------
14:22:16.485 T:560   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <class 'urllib2.URLError'>
                                            Error Contents: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)>
                                            Traceback (most recent call last):
                                              File "C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py", line 799, in <module>
                                                listSpotifyPlaylists(url)
                                              File "C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py", line 331, in listSpotifyPlaylists
                                                content = cache(url, 1)
                                              File "C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py", line 678, in cache
                                                content = opener.open(url).read()
                                              File "C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\Lib\urllib2.py", line 437, in open
                                                response = meth(req, response)
                                              File "C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\Lib\urllib2.py", line 550, in http_response
                                                'http', request, response, code, msg, hdrs)
                                              File "C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\Lib\urllib2.py", line 469, in error
                                                result = self._call_chain(*args)
                                              File "C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\Lib\urllib2.py", line 409, in _call_chain
                                                result = func(*args)
                                              File "C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\Lib\urllib2.py", line 656, in http_error_302
                                                return self.parent.open(new, timeout=req.timeout)
                                              File "C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\Lib\urllib2.py", line 431, in open
                                                response = self._open(req, data)
                                              File "C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\Lib\urllib2.py", line 449, in _open
                                                '_open', req)
                                              File "C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\Lib\urllib2.py", line 409, in _call_chain
                                                result = func(*args)
                                              File "C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\Lib\urllib2.py", line 1240, in https_open
                                                context=self._context)
                                              File "C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\Lib\urllib2.py", line 1197, in do_open
                                                raise URLError(err)
                                            URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:590)>
                                            -->End of Python script error report<--
14:22:16.491 T:560    INFO: Python script stopped
14:22:16.491 T:560   DEBUG: Thread LanguageInvoker 560 terminating
14:22:16.496 T:5224   DEBUG: ------ Window Init (DialogNotification.xml) ------
14:22:16.522 T:5224   ERROR: XFILE::CDirectory::GetDirectory - Error getting plugin://plugin.video.spotitube/?limit&mode=listSpotifyPlaylists&type&url=http%3A%2F%2Fapi.tunigo.com%2Fv3%2Fspace%2Ftoplists%3Fregion%3Dus%26page%3D0%26per_page%3D50%26platform%3Dweb
14:22:16.524 T:5224   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.spotitube/?limit&mode=listSpotifyPlaylists&type&url=http%3A%2F%2Fapi.tunigo.com%2Fv3%2Fspace%2Ftoplists%3Fregion%3Dus%26page%3D0%26per_page%3D50%26platform%3Dweb) failed
14:22:16.524 T:5224   DEBUG: CGUIMediaWindow::GetDirectory (plugin://plugin.video.spotitube/?limit&mode=spotifyMain&type&url)
14:22:16.524 T:5224   DEBUG:   ParentPath = [plugin://plugin.video.spotitube/?content_type=video]
14:22:16.525 T:608   DEBUG: XFILE::CPluginDirectory::StartScript - calling plugin Youtube Music('plugin://plugin.video.spotitube/','4','?limit&mode=spotifyMain&type&url')
14:22:16.525 T:1280   DEBUG: Thread LanguageInvoker start, auto delete: false
14:22:16.525 T:1280    INFO: initializing python engine.
14:22:16.525 T:1280   DEBUG: CPythonInvoker(4, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): start processing
14:22:16.547 T:5940   DEBUG: CAddonDatabase::SetLastUsed[plugin.video.spotitube] took 21 ms
14:22:16.669 T:1280   DEBUG: -->Python Interpreter Initialized<--
14:22:16.669 T:1280   DEBUG: CPythonInvoker(4, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): the source file to load is "C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py"
14:22:16.669 T:1280   DEBUG: CPythonInvoker(4, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): setting the Python path to C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\DLLs;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\Lib;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\python27.zip;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\lib\plat-win;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\lib\lib-tk;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\lib\site-packages
14:22:16.669 T:1280   DEBUG: CPythonInvoker(4, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): entering source directory C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube
14:22:16.669 T:1280   DEBUG: CPythonInvoker(4, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): instantiating addon using automatically obtained id of "plugin.video.spotitube" dependent on version 2.1.0 of the xbmc.python api
14:22:17.025 T:5224   DEBUG: ------ Window Init (DialogBusy.xml) ------
14:22:17.040 T:1280    INFO: CPythonInvoker(4, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): script successfully run
14:22:17.056 T:1280    INFO: Python script stopped
14:22:17.056 T:1280   DEBUG: Thread LanguageInvoker 1280 terminating
14:22:17.064 T:5012   DEBUG: Thread BackgroundLoader start, auto delete: false
14:22:17.073 T:5224   DEBUG: ------ Window Deinit (DialogBusy.xml) ------
14:22:17.081 T:5012   DEBUG: Thread BackgroundLoader 5012 terminating
```

**Comment by tyl0re** [comment]
Since i Dont habve the Error its Hard to Debug. Made a New Version could you try

**Comment by Rouzax** [comment]
Do I need to install it manually or is it uploaded to the repository and installed automatically?

**Comment by Rouzax** [comment]
Is the new version 1.0.11? If so it stil doesn't work.
Will post the debug logs tomorrow.

**Comment by tyl0re** [comment]
Yes 1.0.11 Is the the the newest Version.

The Logfile Would Help

**Comment by Rouzax** [comment]
On Kodi 17 Beta 6 with the 1.0.11 it seems that it now works again. 

Here is the debug log:
```
08:58:32.223 T:5800   DEBUG: CGUIMediaWindow::GetDirectory (plugin://plugin.video.spotitube/?limit&mode=listSpotifyPlaylists&type&url=http%3A%2F%2Fapi.tunigo.com%2Fv3%2Fspace%2Ftoplists%3Fregion%3Dnl%26page%3D0%26per_page%3D50%26platform%3Dweb)
08:58:32.223 T:5800   DEBUG:   ParentPath = [plugin://plugin.video.spotitube/?limit&mode=spotifyMain&type&url]
08:58:32.227 T:5004   DEBUG: XFILE::CPluginDirectory::StartScript - calling plugin Youtube Music('plugin://plugin.video.spotitube/','7','?limit&mode=listSpotifyPlaylists&type&url=http%3A%2F%2Fapi.tunigo.com%2Fv3%2Fspace%2Ftoplists%3Fregion%3Dnl%26page%3D0%26per_page%3D50%26platform%3Dweb')
08:58:32.227 T:1136   DEBUG: Thread LanguageInvoker start, auto delete: false
08:58:32.227 T:1136    INFO: initializing python engine.
08:58:32.345 T:1136   DEBUG: CPythonInvoker(7, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): start processing
08:58:32.467 T:1136   DEBUG: -->Python Interpreter Initialized<--
08:58:32.467 T:1136   DEBUG: CPythonInvoker(7, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): the source file to load is "C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py"
08:58:32.497 T:1856   DEBUG: CAddonDatabase::SetLastUsed[plugin.video.spotitube] took 87 ms
08:58:32.499 T:1136   DEBUG: CPythonInvoker(7, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): setting the Python path to C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\DLLs;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\Lib;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\python27.zip;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\lib\plat-win;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\lib\lib-tk;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python;C:\Program Files\WindowsApps\XBMCFoundation.Kodi_16.9.809.0_x86__4n2hpmxwrvr6p\system\python\lib\site-packages
08:58:32.499 T:1136   DEBUG: CPythonInvoker(7, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): entering source directory C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube
08:58:32.499 T:1136   DEBUG: CPythonInvoker(7, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): instantiating addon using automatically obtained id of "plugin.video.spotitube" dependent on version 2.1.0 of the xbmc.python api
08:58:32.724 T:5800   DEBUG: ------ Window Init (DialogBusy.xml) ------
08:58:32.728 T:5800   DEBUG: ------ Window Init (Pointer.xml) ------
08:58:32.758 T:1136    INFO: CPythonInvoker(7, C:\Users\USER\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): script successfully run
08:58:32.761 T:1136    INFO: Python script stopped
08:58:32.761 T:1136   DEBUG: Thread LanguageInvoker 1136 terminating
08:58:32.781 T:5800   DEBUG: RetrieveMusicInfo() took 0 msec
08:58:32.807 T:4984   DEBUG: Thread BackgroundLoader start, auto delete: false
08:58:32.829 T:5800   DEBUG: ------ Window Deinit (DialogBusy.xml) ------
08:58:33.147 T:4984   DEBUG: Thread BackgroundLoader 4984 terminating
08:58:33.315 T:5612   DEBUG: CTextureCacheJob::GetImageHash - unable to stat url http://d3rt1990lpmkn.cloudfront.net/300/https://u.scdn.co/images/pl/default/eb4767a689d87e109ba06990e63b188c31cd4aa2
08:58:33.316 T:5004   DEBUG: CTextureCacheJob::GetImageHash - unable to stat url http://d3rt1990lpmkn.cloudfront.net/300/https://u.scdn.co/images/pl/default/0a5dcb7f171bef90ae4cf329210e73949961fdd5
08:58:33.325 T:5984   DEBUG: CTextureCacheJob::GetImageHash - unable to stat url http://d3rt1990lpmkn.cloudfront.net/300/https://u.scdn.co/images/pl/default/6fe316b17dd5fd6ef28cb1ae060dc35a175fcf16
08:58:33.333 T:5792   DEBUG: CTextureCacheJob::GetImageHash - unable to stat url http://d3rt1990lpmkn.cloudfront.net/300/https://u.scdn.co/images/pl/default/3af0849eb40064bbee57f9d0edd93fc19c07d08d
08:58:33.407 T:5612   DEBUG: CTextureCacheJob::GetImageHash - unable to stat url http://d3rt1990lpmkn.cloudfront.net/300/https://u.scdn.co/images/pl/default/36226ccd2d4ed6090544d2a82fc11290fba7ccbf
08:58:33.417 T:5004   DEBUG: CTextureCacheJob::GetImageHash - unable to stat url http://d3rt1990lpmkn.cloudfront.net/300/https://u.scdn.co/images/pl/default/f9531b2d95a54b4412b9d078e8ca2bf367b39ea7
08:58:33.418 T:5792   DEBUG: CTextureCacheJob::GetImageHash - unable to stat url http://d3rt1990lpmkn.cloudfront.net/300/https://u.scdn.co/images/pl/default/fb3244955de53a036dc1d70cb4578df504124cb9
08:58:33.431 T:5984   DEBUG: CTextureCacheJob::GetImageHash - unable to stat url http://d3rt1990lpmkn.cloudfront.net/300/https://u.scdn.co/images/pl/default/42ad0fcdf1410179c53a3f35cfec093f447aaa86
08:58:33.496 T:5612   DEBUG: CTextureCacheJob::GetImageHash - unable to stat url http://d3rt1990lpmkn.cloudfront.net/300/https://u.scdn.co/images/pl/default/b77ac22b5a3e9a9f637192c6f4611f370eace2ce
08:58:33.504 T:5792   DEBUG: CTextureCacheJob::GetImageHash - unable to stat url http://d3rt1990lpmkn.cloudfront.net/300/https://u.scdn.co/images/pl/default/b24a4d9b8cf4611e659e1895ebca9e4356106612
08:58:33.508 T:5004   DEBUG: CTextureCacheJob::GetImageHash - unable to stat url http://d3rt1990lpmkn.cloudfront.net/300/https://u.scdn.co/images/pl/default/0c855f35e5c3bd5bb52900599bd9828770a3062e
08:58:33.521 T:5984   DEBUG: CTextureCacheJob::GetImageHash - unable to stat url http://d3rt1990lpmkn.cloudfront.net/300/https://u.scdn.co/images/pl/default/33f1156eaff9def83bd4908daad8502075c04cca
08:58:33.659 T:5984   DEBUG: CTextureCacheJob::GetImageHash - unable to stat url http://d3rt1990lpmkn.cloudfront.net/300/https://u.scdn.co/images/pl/default/36226ccd2d4ed6090544d2a82fc11290fba7ccbf
08:58:33.866 T:5984   DEBUG: CTextureCacheJob::GetImageHash - unable to stat url http://d3rt1990lpmkn.cloudfront.net/300/https://u.scdn.co/images/pl/default/b77ac22b5a3e9a9f637192c6f4611f370eace2ce
08:58:33.874 T:5612   DEBUG: CTextureCacheJob::GetImageHash - unable to stat url http://d3rt1990lpmkn.cloudfront.net/300/https://u.scdn.co/images/pl/default/fb3244955de53a036dc1d70cb4578df504124cb9
08:58:34.211 T:5800   DEBUG: CWinEventsWin32::WndProcWindow is active
```

Will post the debug for Kodi 16 later today.

**Comment by Rouzax** [comment]
In Kodi 16 it still doesn't work.
Here is the Debug log:
```
10:20:14 T:5796   DEBUG: Keyboard: scancode: 0x1c, sym: 0x000d, unicode: 0x000d, modifier: 0x0
10:20:14 T:5796   DEBUG: CInputManager::OnKey: return (0xf00d) pressed, action is Select
10:20:14 T:5796   DEBUG: CGUIMediaWindow::GetDirectory (plugin://plugin.video.spotitube/?limit&mode=listSpotifyPlaylists&type&url=http%3a%2f%2fapi.tunigo.com%2fv3%2fspace%2ftoplists%3fregion%3dnl%26page%3d0%26per_page%3d50%26platform%3dweb)
10:20:14 T:5796   DEBUG:   ParentPath = [plugin://plugin.video.spotitube/?limit&mode=spotifyMain&type&url]
10:20:14 T:5804   DEBUG: XFILE::CPluginDirectory::StartScript - calling plugin Youtube Music('plugin://plugin.video.spotitube/','6','?limit&mode=listSpotifyPlaylists&type&url=http%3a%2f%2fapi.tunigo.com%2fv3%2fspace%2ftoplists%3fregion%3dnl%26page%3d0%26per_page%3d50%26platform%3dweb')
10:20:14 T:5804   DEBUG: XFILE::CPluginDirectory::WaitOnScriptResult - waiting on the Youtube Music (id=13) plugin...
10:20:14 T:5704   DEBUG: Thread LanguageInvoker start, auto delete: false
10:20:14 T:5704    INFO: initializing python engine.
10:20:14 T:5704   DEBUG: CPythonInvoker(13, C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): start processing
10:20:14 T:5704   DEBUG: -->Python Interpreter Initialized<--
10:20:14 T:5704   DEBUG: CPythonInvoker(13, C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): the source file to load is "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py"
10:20:14 T:5704   DEBUG: CPythonInvoker(13, C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): setting the Python path to C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube;C:\Program Files (x86)\Kodi\system\python\DLLs;C:\Program Files (x86)\Kodi\system\python\Lib;C:\Program Files (x86)\Kodi\python27.zip;C:\Program Files (x86)\Kodi\system\python\lib\plat-win;C:\Program Files (x86)\Kodi\system\python\lib\lib-tk;C:\Program Files (x86)\Kodi;C:\Program Files (x86)\Kodi\system\python;C:\Program Files (x86)\Kodi\system\python\lib\site-packages
10:20:14 T:5704   DEBUG: CPythonInvoker(13, C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): entering source directory C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube
10:20:14 T:5704   DEBUG: CPythonInvoker(13, C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): instantiating addon using automatically obtained id of "plugin.video.spotitube" dependent on version 2.1.0 of the xbmc.python api
10:20:15 T:5704   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <type 'exceptions.TypeError'>
                                            Error Contents: __init__() got an unexpected keyword argument 'context'
                                            Traceback (most recent call last):
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py", line 805, in <module>
                                                listSpotifyPlaylists(url)
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py", line 337, in listSpotifyPlaylists
                                                content = cache(url, 1)
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py", line 684, in cache
                                                content = opener.open(url).read()
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\urllib2.py", line 437, in open
                                                response = meth(req, response)
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\urllib2.py", line 550, in http_response
                                                'http', request, response, code, msg, hdrs)
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\urllib2.py", line 469, in error
                                                result = self._call_chain(*args)
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\urllib2.py", line 409, in _call_chain
                                                result = func(*args)
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\urllib2.py", line 656, in http_error_302
                                                return self.parent.open(new, timeout=req.timeout)
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\urllib2.py", line 431, in open
                                                response = self._open(req, data)
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\urllib2.py", line 449, in _open
                                                '_open', req)
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\urllib2.py", line 409, in _call_chain
                                                result = func(*args)
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\urllib2.py", line 1240, in https_open
                                                context=self._context)
                                              File "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\urllib2.py", line 1166, in do_open
                                                h = http_class(host, timeout=req.timeout, **http_conn_args)
                                            TypeError: __init__() got an unexpected keyword argument 'context'
                                            -->End of Python script error report<--
10:20:15 T:5704    INFO: Python script stopped
10:20:15 T:5704   DEBUG: Thread LanguageInvoker 5704 terminating
10:20:15 T:5804   DEBUG:  XFILE::CPluginDirectory::WaitOnScriptResult - plugin exited prematurely - terminating
10:20:15 T:5796   ERROR: XFILE::CDirectory::GetDirectory - Error getting plugin://plugin.video.spotitube/?limit&mode=listSpotifyPlaylists&type&url=http%3a%2f%2fapi.tunigo.com%2fv3%2fspace%2ftoplists%3fregion%3dnl%26page%3d0%26per_page%3d50%26platform%3dweb
10:20:15 T:5796   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.spotitube/?limit&mode=listSpotifyPlaylists&type&url=http%3a%2f%2fapi.tunigo.com%2fv3%2fspace%2ftoplists%3fregion%3dnl%26page%3d0%26per_page%3d50%26platform%3dweb) failed
10:20:15 T:5796   DEBUG: CGUIMediaWindow::GetDirectory (plugin://plugin.video.spotitube/?limit&mode=spotifyMain&type&url)
10:20:15 T:5796   DEBUG:   ParentPath = [plugin://plugin.video.spotitube/]
10:20:15 T:1508   DEBUG: XFILE::CPluginDirectory::StartScript - calling plugin Youtube Music('plugin://plugin.video.spotitube/','7','?limit&mode=spotifyMain&type&url')
10:20:15 T:1508   DEBUG: XFILE::CPluginDirectory::WaitOnScriptResult - waiting on the Youtube Music (id=14) plugin...
10:20:15 T:5716   DEBUG: Thread LanguageInvoker start, auto delete: false
10:20:15 T:5716    INFO: initializing python engine.
10:20:15 T:5716   DEBUG: CPythonInvoker(14, C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): start processing
10:20:15 T:5716   DEBUG: -->Python Interpreter Initialized<--
10:20:15 T:5716   DEBUG: CPythonInvoker(14, C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): the source file to load is "C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py"
10:20:15 T:5716   DEBUG: CPythonInvoker(14, C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): setting the Python path to C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube;C:\Program Files (x86)\Kodi\system\python\DLLs;C:\Program Files (x86)\Kodi\system\python\Lib;C:\Program Files (x86)\Kodi\python27.zip;C:\Program Files (x86)\Kodi\system\python\lib\plat-win;C:\Program Files (x86)\Kodi\system\python\lib\lib-tk;C:\Program Files (x86)\Kodi;C:\Program Files (x86)\Kodi\system\python;C:\Program Files (x86)\Kodi\system\python\lib\site-packages
10:20:15 T:5716   DEBUG: CPythonInvoker(14, C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): entering source directory C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube
10:20:15 T:5716   DEBUG: CPythonInvoker(14, C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): instantiating addon using automatically obtained id of "plugin.video.spotitube" dependent on version 2.1.0 of the xbmc.python api
10:20:15 T:5716   DEBUG: LocalizeStrings: no strings.po file exist at C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\resources\language\English, fallback to strings.xml
10:20:15 T:1508   DEBUG: XFILE::CPluginDirectory::WaitOnScriptResult- plugin returned successfully
10:20:15 T:5716    INFO: CPythonInvoker(14, C:\Users\Kodi\AppData\Roaming\Kodi\addons\plugin.video.spotitube\default.py): script successfully run
10:20:15 T:5796   DEBUG: RetrieveMusicInfo() took 0 msec
10:20:15 T:856   DEBUG: Thread BackgroundLoader start, auto delete: false
10:20:15 T:5796   DEBUG: ------ Window Init (DialogKaiToast.xml) ------
10:20:15 T:5716    INFO: Python script stopped
10:20:15 T:5716   DEBUG: Thread LanguageInvoker 5716 terminating
10:20:15 T:856   DEBUG: Thread BackgroundLoader 856 terminating
10:20:20 T:5796   DEBUG: ------ Window Deinit (DialogKaiToast.xml) ------
```

By the way both Kodi's are installed on a fully patched Windows 10 Enterprise system.

**Comment by Rouzax** [comment]
If you want I can maybe provide you with a VM that has the issue to test with.

**Comment by Rouzax** [comment]
With KODI 17 the list problem is solved however the thumbnail issue is still present. Will create a new ticket for that.
